packages/manager/modules/bm-server-components/src/display-name/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+
+import component from './component';
+
+const moduleName = 'ovhManagerBmServerComponentsDisplayName';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngTranslateAsyncLoader',
```

Comment:
`ngTranslateAsyncLoader` might be removed

Reply:
it's a applied.